WooCommerce/Classes/ViewRelated/JetpackSetup/JetpackSetupCoordinator.swift

```diff
@@ -119,26 +143,142 @@ private extension JetpackSetupCoordinator {
         benefitsController?.present(UINavigationController(rootViewController: viewController), animated: true)
     }
 
+    func startJetpackSetupFlow(authToken: String) {
+        /// Dismiss any existing login flow if possible.
+        if rootViewController.topmostPresentedViewController is LoginNavigationController {
+            rootViewController.dismiss(animated: false)
+        }
+        let progressView = InProgressViewController(viewProperties: .init(title: Localization.pleaseWait, message: ""))
+        rootViewController.topmostPresentedViewController.present(progressView, animated: true)
+        let action = JetpackConnectionAction.fetchJetpackUser { [weak self] result in
+            guard let self else { return }
+            self.rootViewController.dismiss(animated: true)
```

Comment:
@jaclync I added tests in 2a3bf67ef5f4a04ccbd7de12c1ce09b439c31d5c, but the test failed since we are keeping the scheme in the credentials file and I'm not sure if exposing the scheme is a security threat. So, I removed the tests - what would you suggest for this situation?

Reply:
This makes sense - I updated this in 1e79436bfaebd81d5c17e395e213f71f50608f5b.